src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseEntitySelectPage.tsx

```diff
@@ -18,6 +18,22 @@ import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
 import type {QBOReimbursableExportAccountType} from '@src/types/onyx/Policy';
 
+function Footer({isTaxEnabled, isLocationsEnabled}: {isTaxEnabled: boolean; isLocationsEnabled: boolean}) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    if (!isTaxEnabled && !isLocationsEnabled) {
+        return null;
+    }
+
+    return (
+        <View style={[styles.flex1, styles.alignItemsCenter, styles.gap2, styles.mt1]}>
```

Comment:
Oh! On native platforms (i.e. android and iOS), hint texts are not displayed in footer. This one got slipped throughout reviews.
But overall the code looks good and tests well for all the other issues discussed above. 
Will complete the checklist as soon as this is done.

https://github.com/Expensify/App/assets/3004659/a1f82cec-e820-4cc8-8727-9401c84a18d8

https://github.com/Expensify/App/assets/3004659/ad28484b-3785-467b-8172-9be8dfca1804

Reply:
Hm, why is there an additional screen on native?

Reply:
^^ scratch that. We kept it here because there's typically an `Account` || `Accounts payable` row to select in addition. So more to the point, why is that seemingly missing here? Comparing the video of desktop in the OP: 

<img width="800" alt="image" src="https://github.com/Expensify/App/assets/16232057/91933062-2166-4b8b-a6db-70bd91a54933">

Reply:
Alright yeah, as soon as I choose to import locations the `Account` row disappears. 

https://github.com/Expensify/App/assets/16232057/2f1a67ab-fb8b-418b-a890-da4d63520ec1

Note: When testing this build the push row label is reading `Accounts payable` when it should be `Account` for Journal Entry export, but @s77rt had cleaned that up here https://github.com/Expensify/App/pull/42548, so not sure  if those changes aren't properly reflected in the build for this PR.

Reply:
This PR is older than https://github.com/Expensify/App/pull/42548 thus it does not have the change. Merging `qbo-freeze-branch` into this PR should reflect the changes correctly.

Reply:
Cool, so that explains that one. Yet there's still a bug in this PR that is hiding it when locations are imported. Is that correct?

Reply:
> Yet there's still a bug in this PR that is hiding it when locations are imported. Is that correct?

Yes. That's correct. And we need to fix this here itself.

Reply:
Ah that's a bug. If locations are enabled we disable the Check and Vendor bill options but keep the journal entry option thus we should have a row to update it.

Current logic:
- Show the account row if location is not enabled

Correct logic:
- Show the account row if location is not enabled or if the taxes are not enabled


Note: If taxes are enabled then the journal entry becomes unavailable too

Reply:
Hm, I'm not sure I'm following. Why wouldn't `Account` just be conditional on if `Journal Entry` is selected for export or not? :/

Reply:
If `Journal Entry` is selected and Taxes are enabled then we'd want to show an error message `Journal entry is not available when taxes enabled. please select a different export option.` (and in this case the `Account` row should not be visible)

Reply:
`Journal entry` being selected should be the driver of the `Account` field showing. So if JE is selected, we should show `Account`. 

I don't think we need to tie the `Locations` or `Taxes` logic to the `Account` || `Accounts payable` || `Bank account` field showing as well do we? :/

Reply:
thank you @s77rt for the PR 🙇 

going back to the original issue reported by @rojiphil, do you know why the footer component is not displayed on `QuickbooksOutOfPocketExpenseEntitySelectPage` on mobile?

Reply:
> do you know why the footer component is not displayed on `QuickbooksOutOfPocketExpenseEntitySelectPage` on mobile?

Looks like a styling issue. The following seems to work. What do you think?
```suggestion
        <View style={[styles.flexColumn, styles.justifyContentCenter, styles.gap2, styles.mt1]}>
```

Reply:
style fix https://github.com/Expensify/App/pull/42921

Reply:
thank you for the fix 🙇